src/background.js

```diff
@@ -1074,6 +1078,18 @@ function onMessageHandler(request, sender, callback) {
 		});
 		return true;
 	}
+	if (name === 'promoModals.sawPlusPromo') {
+		promoModals.recordPlusPromoSighting();
+		return true;
```

Comment:
We should return `false` here since this is synchronous and there is no `callback`

Reply:
Done